README.md

```diff
@@ -147,9 +147,14 @@ https://reporter.app.lumogon.com/UuokYc0VMgA4oAZYaJRaN6O7xMqakDLsqgHGs0dBmJY=
 
 ### More ways to use Lumogon
 
+The [examples](examples/README.md) directory provides more thorough examples of
+how to use the data provided by Lumogon to test and build your images and containers.
+
+### Lumogon command options
+
 Let's get the usage for the Lumogon client:
 
-``` shell
+```
```

Comment:
Dropped the shell syntax because it causes unintentional shell commands in the help output to highlight. Otherwise, it's identical formatting to generic code formatting.